### PR TITLE
Login & signup

### PR DESCRIPTION
**Pull request by Creator-singh**
- Login Page
 login_page

 - Signup Page
 signup_page

### CODE REVIEW

**Review by hiteshverma27 — APPROVED** [review]
Looks good

File: styles/login-signup.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.wrapper{
+    display: flex;
```
Comment (aniketsingh13) [inline styles/login-signup.css:2]: you can use utility here

File: styles/login-signup.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    justify-content: center;
+    margin: 5rem 0;
+}
+
+.form{
+    width: 40rem;
+    height: fit-content;
+    border: 2px solid var(--secondary-clr);
+    padding: 2rem;
+    margin: 1rem;
```
Comment (aniketsingh13) [inline styles/login-signup.css:13]: here also

**Review by aniketsingh13 — APPROVED** [review]
other thing look good to go

**Pull request merged**